includes/razorpay-webhook.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+require_once __DIR__.'/../razorpay-payments.php';
+
+require_once __DIR__.'/../razorpay-sdk/Razorpay.php';
+use Razorpay\Api\Api;
+
+class RZP_Webhook
+{
+    protected $razorpay;
+    protected $api;
+
+    function __construct()
+    {
+        $this->razorpay = new WC_Razorpay();
+
+        $this->api = $this->razorpay->getRazorpayApiInstance();
+    }
+
+    public function process()
+    {
+        $post = file_get_contents('php://input');
+
+        $data = json_decode($post, true);
```

Comment:
@captn3m0 Handled. Tested. 
razorpay-webhook.php - process function. 
We check for whether data has event set.